src/answers-umd.js

```diff
@@ -514,14 +514,29 @@ class Answers {
   }
 
   /**
-   * Processes a translation which includes performing interpolation, pluralization, or both
-   * @param {string} translations The translations, or a stringified JSON of possible translations
+   * Processes a translation which includes performing interpolation, pluralization, or
+   * both
+   * @param {string | Object} translations The translation, or an object containing
+   * translated plural forms
    * @param {Object} interpolationParams Params to use during interpolation
    * @param {number} count The count associated with the pluralization
+   * @param {string} language The langauge associated with the pluralization
    * @returns {string} The translation with any interpolation or pluralization applied
    */
-  processTranslation (translations, interpolationParams, count) {
-    return TranslationProcessor.process(translations, interpolationParams, count);
+  processTranslation (translations, interpolationParams, count, language) {
+    const initLocale = this._getInitLocale();
+    language = language || initLocale.substring(0, 2);
```

Comment:
Are there plural forms specific to locales? Does the library we pulled in support that at all? I'm wondering if we want to be stripping the suffix from the locale here.

Reply:
The library we pulled in only supports language codes. It supports some three-character language codes, but we don't use any of those so Rose and I agreed that using the two-character language code was sufficient.